File: hw/astudillo/U3/HW20Template/Template/src/main/java/ec/edu/espe/template/template/model/BeverageTest.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.template.template.model;

import ec.edu.espe.template.template.model.Coffe;
import ec.edu.espe.template.template.model.Tea;

/**
 *
 * @author PC
 */
public class BeverageTest {
    public static void main(String[] args){
        Tea tea = new Tea();
        Coffe coffe = new Coffe();
        
        System.out.println("Making Tea ...");
        tea.prepareRecipe();
        
        System.out.println("\nMaking coffee ..." ); 
        coffe.prepareRecipe();
    }
}

File: hw/paillacho/U3/HW21TemplateMethod/TemplateMethod/python_version/template/model/beverage_test.py
import sys
import os
sys.path.append(os.path.dirname(os.path.abspath(__file__)))

from tea import Tea
from coffee import Coffee


class BeverageTest:
    
    @staticmethod
    def main():
        tea = Tea()
        coffee = Coffee()
        
        print("Making Tea ...")
        tea.prepare_recipe()
        
        print("\nMaking coffee ...")
        coffee.prepare_recipe()


if __name__ == "__main__":
    BeverageTest.main()
